src/features/user-stats/model/index.ts

```diff
@@ -1,2 +1,3 @@
 export { useUserStats } from './use-user-stats';
 export type { MathGameResult } from './types';
+export type { SpeedCountingGameResult } from './types';
```

Comment:
there is 2 imports from the same source already.
do `export type { MathGameResult, SpeedCountingGameResult } from './types';`